Sources/Data/Services/ExperienceStore/ExperienceStore.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  ExperienceStore.swift
+//  RoverData
+//
+//  Created by Sean Rucker on 2018-05-04.
+//  Copyright © 2018 Rover Labs Inc. All rights reserved.
+
+import Foundation
+
+public protocol ExperienceStore {
```

Comment:
I'd suggest just making this a class instead of a protocol/implementation pair. One of the learnings I took from SDK 2.x is that this pattern makes it harder for developers to customize services. The counter would be that it makes it more testable, but I'd prioritize developer experience over our own test cases in priority.